platforms/BUILD

```diff
@@ -75,3 +75,39 @@ platform(
     },
     parents = [":cortex_m"],
 )
+
+platform(
+    name = "stm32f2xx",
+    parents = [":cortex_m3"],
+    visibility = ['//visibility:public'],
```

Comment:
Would you mind removing these visibility lines, and adding;
```package(default_visibility = ["//visibility:public"])```
To the top of this file, it makes sense for all of these targets to be public.